WordPress/Classes/ViewRelated/Blog/Onboarding Questions Prompt/OnboardingQuestionsCoordinator.swift

```diff
@@ -0,0 +1,121 @@
+import Foundation
+import UIKit
+
+enum OnboardingOption: String {
+    case stats = "stats"
+    case writing = "writing"
+    case notifications = "notifications"
+    case reader = "reader"
+    case showMeAround = "show_me_around"
+    case skip = "skip"
```

Comment:
[nit]

```suggestion
    case stats
    case writing
    case notifications
    case reader
    case showMeAround = "show_me_around"
    case skip
```

Reply:
Yep, that's much better. Done in: ba3d55e9d5e3c7acd752aea8130b00ac8835807b